hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/kafka/KafkaIntegrationSqlTest.java

```diff
@@ -41,20 +41,18 @@ public void should_read_from_imap_to_kafka() {
         createMap(mapName);
 
         executeSql("INSERT INTO " + mapName + " VALUES\n" +
-                   "  (1, 'ABCD', 5.5, 10),\n" +
-                   "  (2, 'EFGH', 14, 20);");
+                "  (1, 'ABCD', 5.5, 10),\n" +
+                "  (2, 'EFGH', 14, 20);");
 
         assertMapContents(mapName, Map.of(
                 1, new HazelcastJsonValue("{\"ticker\":\"ABCD\",\"price\":\"5.5\",\"amount\":10}"),
                 2, new HazelcastJsonValue("{\"ticker\":\"EFGH\",\"price\":\"14\",\"amount\":20}"))
         );
 
         executeSql("CREATE JOB testJob\n" +
-                   "OPTIONS (\n" +
-                   "  'processingGuarantee' = 'exactlyOnce'\n" +
-                   ") AS\n" +
-                   "SINK INTO " + topicName + "\n" +
-                   "SELECT __key, ticker, price, amount FROM " + mapName);
+                "AS\n" +
```

Comment:
this will no longer be a test for HZ-2726

Reply:
Should I remove it @ldziedziul ?

Reply:
I would leave it for the reasons @k-jamroz brought up